src/olmo_core/data/__init__.py

```diff
@@ -56,6 +56,7 @@
     "NumpyDatasetDType",
     "TokenizerConfig",
     "TokenizerName",
+    "BaseDataMix",
```

Comment:
I think you're missing the actual import above

Reply:
Ah, fixed